src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
> Actually it's executePendingBrowserWindowCommandCommand

To be clear it should be `executeExecutePendingBrowserWindowCommand` in this case since we have the `ExecutePendingBrowserWindowCommand` command and here we call to execute it. Maybe `runExecutePendingBrowserWindowCommand` or something like this